src/axom/quest/tests/CMakeLists.txt

```diff
@@ -156,14 +156,20 @@ if(ENABLE_MPI AND MFEM_FOUND AND MFEM_USE_MPI
             DEPENDS_ON ${quest_tests_depends} mfem
             FOLDER axom/quest/tests
         )
-
-        set(_numMPITasks 1)
-        axom_add_test(
-            NAME ${test_name}
-            COMMAND ${test_name}_test
-            NUM_MPI_TASKS ${_numMPITasks}
-        )
     endforeach()
+
+    set(_numMPITasks 1)
+    axom_add_test(NAME quest_intersection_shaper
+        COMMAND quest_intersection_shaper_test
+        NUM_MPI_TASKS ${_numMPITasks}
+        NUM_OMP_THREADS 8 # limit omp parallelism
```

Comment:
@BradWhitlock -- I limited the intersection-based shaping tests to  8 threads. This test was causing occasional hangs since it was occasionally using more than the available number of threads.

In my debug config, this roughly doubles the runtime for this test from ~40s to ~80s (give or take).

I noticed that the internal refinement is set to 7 yielding meshes with tens of thousands of octahedra (specifically, 382 octahedra per segment). 

Could/should we reduce the level for our general tests to something like 4 or 5 (and update the baselines), with the understanding that the results will be somewhat less accurate?

Reply:
I think this was the cause of the hang that @white238 mentioned at today's meeting

Reply:
Reducing the refinement level would be a good way to speed up the intersection tests. We might need to rebaseline then.

Reply:
Thanks -- I created https://github.com/LLNL/axom/issues/1136 to track this